overlord/servicestate/quota_control.go

```diff
@@ -83,6 +84,58 @@ func quotaGroupsAvailable(st *state.State) error {
 	return nil
 }
 
+func isQuotaAvailable(st *state.State, feature features.SnapdFeature, settingName string) error {
+	tr := config.NewTransaction(st)
+	status, err := features.Flag(tr, feature)
+	if err != nil && !config.IsNoOption(err) {
+		return err
+	}
+	if !status {
+		return fmt.Errorf("experimental feature disabled - test it by setting 'experimental.%s' to true", settingName)
```

Comment:
(nitpick/bikesheed) maybe use s/test/use/ ? Like: `experimental feature disabled - use it by setting 'experimental.%s` ?

Reply:
we use 'test' other places as well, so maybe this is a job for a follow up to align them and change to 'use'?